chainercv/functions/ps_roi_average_pooling_2d.py

```diff
@@ -0,0 +1,422 @@
+# Modified work:
+# ------------------------------------------------------------------------
+# Copyright (c) 2018 Preferred Networks, Inc.
+# ------------------------------------------------------------------------
+
+# Original works of CUDA kernel in forward_gpu and backward_gpu:
+# ------------------------------------------------------------------------
+# Copyright (c) 2017 Microsoft
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Written by Yi Li, Tairui Chen, Guodong Zhang, Haozhi Qi and Jifeng Dai
+# https://github.com/msracver/FCIS
+# ------------------------------------------------------------------------
+
+
+from __future__ import division
+
+import numpy as np
+import six
+
+import chainer
+from chainer.backends import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+def _pair(x):
```

Comment:
The name of this helper should be improved.

Reply:
rename to `_outsize`